ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -146,6 +147,26 @@ export const ProfilePage = fp.flow(
         updating: false
       };
     }
+    static displayName = 'ProfilePage';
+
+    saveProfileWithRenewal = withSuccessModal({
```

Comment:
On save, call the existing function, then redirect on success

Reply:
these changes work nicely